UnitTests/OIDTokenRequestTests.m

```diff
@@ -126,7 +149,8 @@ + (OIDTokenRequest *)testInstanceRefresh {
                                               scopes:scopesArray
                                         refreshToken:kRefreshTokenTestValue
                                         codeVerifier:authResponse.request.codeVerifier
-                                additionalParameters:additionalParameters];
+                                additionalParameters:additionalParameters
+                                   additionalHeaders:additionalHeaders];
```

Comment:
Please add a check for `additionalHeaders` in `testCopying` and `testSecureCoding`.